terraform/applications/discourse/cdn.tf

```diff
@@ -12,6 +12,19 @@ resource "aws_cloudfront_distribution" "discourse" {
 
   origin {
     domain_name = var.discourse_cdn_zone
+    origin_id   = "discourse-pull-origin-old"
+    origin_path = ""
+
+    custom_origin_config {
+      http_port              = "80"
+      https_port             = "443"
+      origin_protocol_policy = "https-only" # Only talk to the origin over HTTPS
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```

Comment:
Same comment here but having the extra protocols isn't going to hurt anything it'll default to 1.2 in this case.